### ISSUE TITLE
Variables not exist in double quota should raise error

### ISSUE THREAD

**Issue by xiaojueguan** [nvuillam/npm-groovy-lint#255]
### problem description:
File with below content should raise an error, as the variable Name not exist in the context but not
```
def name = 'xiaojue'
println("${Name}")
```
reproduce action:
1. mkdir a folder groovy
2. cd groovy and create a file with content like below
```
def name = 'xiaojue'
println("${Name}")
```
3. run below content
```
docker run --rm -u "$(id -u):$(id -g)" -w=/tmp -v "$PWD":/tmp nvuillam/npm-groovy-lint
```

**Comment by nvuillam** [comment]
@xiaojueguan npm-groovy-lint embeds CodeNarc but does not have its own rules

So if you want a new rule, you can request it in CodeNarc repo :) https://github.com/CodeNarc/CodeNarc
